### PR TITLE
ACRS 110 Referral submitted page

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What? 
[ACRS-110](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-110)DECLARATION & SUBMIT - Referral submitted page

## Why? 
To directed users to the referral submitted page and confirm application is submitted

## How? 
- Added html page to render the header and text in page 
- Added behaviour to capture the decision email when user want to receive decision in different email 
 
## Testing?
Tested manually

## Screenshots (optional)
Screenshot 2024-06-12 at 16 31 29

## Check list

- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant)
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging
- [ ]

### CODE REVIEW

File: apps/acrs/index.js [hunk 1]
```diff
@@ -622,8 +623,8 @@ module.exports = {
       next: '/decision-postal-address'
     },
     '/email-decision': {
-      behaviours: SaveFormSession,
-      fields: ['is-decision-by-email', 'is-decision-by-email-detail'],
+      behaviours: [SaveFormSession, SaveDecisionEmail],
+      fields: ['is-decision-by-email', 'is-decision-by-email-detail', 'decision-email'],
```
Comment (mislam987) [inline apps/acrs/index.js:627]: Why is the `decision-email` field needed given that it is visually hidden in `/fields/index.js`?

File: apps/acrs/views/content/referral-submitted.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+We have sent a confirmation email to <strong> {{values.decision-email}} </strong>
+
+<br>
```
Comment (mislam987) [inline apps/acrs/views/content/referral-submitted.md:3]: Don't think the <br> tags are needed in markdown files, unless it's after a bullet point list; blank lines should be sufficient. Try without the tags and see if it looks okay.

File: apps/acrs/behaviours/get-decision-email.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    const save = super.saveValues(req, res, next);
+    const email = req.sessionModel.get('user-email');
+    if (req.form.values['is-decision-by-email'] === 'yes') {
+      req.sessionModel.set('decision-email', email);
+    } else {
+      req.sessionModel.set('decision-email', req.form.values['is-decision-by-email-detail']);
+    }
+    return save;
+  }
+};
```
Comment (mislam987) [inline apps/acrs/behaviours/get-decision-email.js:13]: This behaviour looks identical to the `save-decision-email.js` behaviour. Are they both needed?

Comment (TemitopeAyokuHO) [reply apps/acrs/behaviours/get-decision-email.js]: Removed

File: apps/acrs/views/content/referral-submitted.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+<h2>What happens next</h2>
+You do not need to do anything now. We will contact you with your decision, or to ask for more information.
+
+<br>
+<h2>If you need to provide evidence later </h2>
+You can email extra information or evidence to <a>example@homeoffices.gov.uk</a>.
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply apps/acrs/views/content/referral-submitted.md]: Yes done

File: apps/acrs/index.js (hunk 1 above)
Comment (TemitopeAyokuHO) [reply apps/acrs/index.js]: removed not needed

**Review by mislam987 — APPROVED** [review]
Looks okay to me.

**Pull request merged**